climpred/prediction.py

```diff
@@ -28,6 +35,81 @@
 )
 
 
+def get_metric_comparison_dim(metric, comparison, dim, kind):
+    """Check whether `metric`, `comparison` and `dim` can work together in experiment
+    type `kind` and return corrected.
+
+    Args:
+        metric (str): metric or alias string
+        comparison (str): Description of parameter `comparison`.
+        dim (list of st or str): dimension to apply metric to.
+        kind (str): experiment type from ['hindcast', 'PM'].
+
+    Returns:
+        metric (Metric): metric class
+        comparison (Comparison): comparison class.
+        dim (list of st or str): corrected dimension to apply metric to.
+    """
+    # check kind allowed
+    is_in_list(kind, ['hindcast', 'PM'], str)
```

Comment:
I think `str` should be `'kind'`. Current error message if you don't use 'hindcast' or 'PM' would be:

```python-traceback
KeyError: "Specify <class 'str'> from ['hindcast', 'PM']: got foo"
```

Updated would be:
```python-traceback
KeyError: "Specify kind from ['hindcast', 'PM']: got foo"
```